src-plugins/medSegmentation/msegAlgorithmPaintToolbox.cpp

```diff
@@ -666,6 +668,7 @@ void AlgorithmPaintToolbox::import()
 {
     setOutputMetadata(m_imageData, m_maskData);
     medDataManager::instance()->importData(m_maskData, true);
+    maskHasBeenSaved = true;
```

Comment:
Ok so I completely misread the code...
Apparently I got the same syndrome as Mike last time.
My bad.